project-base/src/Shopsys/ShopBundle/Resources/views/Front/Content/Error/error.html.twig

```diff
@@ -39,6 +39,10 @@
                     <p>
                         {{ 'Please excuse this error, we are working on fixing it. If needed, contact us on %mail% or by phone %phone%.'|trans({ '%mail%': getShopInfoEmail(), '%phone%': getShopInfoPhoneNumber() }) }}
                     </p>
+
+                    <p>
+                        Error ID: <span id="js-error-id">{{ '{{ERROR_ID}}' }}</span>
```

Comment:
this identifier is for acceptance test only - it has no other purpose

Reply:
🤔 what about `unique-error-id`?

Reply:
We are using `js-` prefix to identifiers or classes to avoid to be replaced or removed by any coders as it is required for some our functional things.